MH4U Dex/CoreDataController.m

```diff
@@ -177,6 +210,50 @@ - (void)loadMonsterDropData
     }
 }
 
+- (void)loadQuestData
+{
+    NSArray *questList = [self loadArrayFromJsonFileNamed:QuestsFileName];
+    
+    for (NSDictionary *questDict in questList) {
+        Quest *quest;
+        NSNumber *questID = questDict[QuestIDKey];
+        NSPredicate *questPredicate = [NSPredicate predicateWithFormat:@"%K == %d", QuestAttributes.id, questID.intValue];
```

Comment:
It would be better to use `questID` and `%@`—using `.intValue` and `%d` limits the range of values that will work more than using the `NSNumber` object directly.